backend/compact-connect/lambdas/common-python/cc_common/data_model/schema/provider.py

```diff
@@ -62,7 +65,14 @@ class ProviderRecordSchema(CalculatedStatusRecordSchema, ProviderPublicSchema):
     # Generated fields
     privilegeJurisdictions = Set(String, required=False, allow_none=False, load_default=set())
     providerFamGivMid = String(required=False, allow_none=False, validate=Length(2, 400))
-    providerDateOfUpdate = Date(required=True, allow_none=False)
+    providerDateOfUpdate = DateTime(required=True, allow_none=False)
+
+    @pre_load
+    def pre_load_initialization(self, in_data, **kwargs):  # noqa: ARG001 unused-argument
+        in_data = super().pre_load_initialization(in_data, **kwargs)
+        in_data['providerDateOfUpdate'] = ensure_value_is_datetime(in_data['providerDateOfUpdate'])
+
```

Comment:
So the reason I added the super call was because it was not working without it. Although reading through your link I'm realizing it probably wasn't working because I was using the same method name, which was causing the base class method to be overridden 🤦 I'm going to rename the base class method to `ensure_date_of_update_is_datetime`

Reply:
Ah, yeah! That would do it. Sounds like a plan.